src/app/pages/send/send-crypto-asset-form/components/recipient-accounts-drawer/recipient-accounts-drawer.tsx

```diff
@@ -19,24 +19,30 @@ export const RecipientAccountsDrawer = memo(() => {
 
   const onGoBack = useCallback(() => navigate('..', { replace: true }), [navigate]);
 
-  return accounts ? (
+  if (!accounts) return null;
+
+  const accountsExcludingCurrent = accounts;
```

Comment:
why not just `const accountsExcludingCurrent = useStacksAccounts();`

Reply:
will change, I'd written logic to exclude existing account from list, but it broke all the tests